types/datum.go

```diff
@@ -294,7 +294,7 @@ func (d *Datum) SetMysqlEnum(b Enum, collation string) {
 
 // GetMysqlSet gets Set value
 func (d *Datum) GetMysqlSet() Set {
-	str := string(hack.String(d.b))
+	str := string(d.b)
```

Comment:
Datum.GetBytes returns reference either.
Better to modify the code in aggregation parts.